blocks/library/embed/index.js

```diff
@@ -19,6 +19,7 @@ import { registerBlockType, query } from '../../api';
 import Editable from '../../editable';
 import BlockControls from '../../block-controls';
 import BlockAlignmentToolbar from '../../block-alignment-toolbar';
+import BlockDescription from '../../block-description';
```

Comment:
Argh how embarrassing. This was copy/pasta — which I'm not sure how happened. Great catch, thanks for this!